public/app/plugins/panel/xychart/module.tsx

```diff
@@ -3,11 +3,11 @@ import { commonOptionsBuilder } from '@grafana/ui';
 
 import { AutoEditor } from './AutoEditor';
 import { ManualEditor } from './ManualEditor';
-import { XYChartPanel2 } from './XYChartPanel2';
+import { XYChartPanel } from './XYChartPanel';
 import { getScatterFieldConfig } from './config';
 import { Options, FieldConfig, defaultFieldConfig } from './panelcfg.gen';
 
-export const plugin = new PanelPlugin<Options, FieldConfig>(XYChartPanel2)
+export const plugin = new PanelPlugin<Options, FieldConfig>(XYChartPanel)
```

Comment:
we should modify Tooltip mode to only support `Single` and `Hidden` (see modifying the `addTooltipOptions` func)